application/controllers/admin/Tokens.php

```diff
@@ -839,7 +838,14 @@ public function edit($iSurveyId, $iTokenId, $ajax = false)
                 foreach ($aAdditionalAttributeFields as $attr_name => $desc) {
                     $value = $request->getPost($attr_name, '');
                     if ($desc['mandatory'] == 'Y' && trim($value) == '') {
-                        $sOutput .= sprintf(gT("Notice: Field '%s' was left empty, even though it is a mandatory attribute."), $desc['description']) . '<br>';
+                        /* All this part is disable via JS, no way to submit : issue #19548 https://bugs.limesurvey.org/view.php?id=19548*/
```

Comment:
What do you mean by disable?

Reply:
> What do you mean by disable?

I put the number of mantis issue : https://bugs.limesurvey.org/view.php?id=19548

Here it's the PHP part : allow to save empty value in required attribute : but a JS function disable this.